stack/__main__.py

```diff
@@ -0,0 +1,391 @@
+"""
+Pulumi stack to set up buckets and permission groups
+"""
+
+import pulumi
+import pulumi_gcp as gcp
+
+DOMAIN = 'populationgenomics.org.au'
+CUSTOMER_ID = 'C010ys3gt'
```

Comment:
Another is this okay to be public facing.

Reply:
Ditto.